docs/quickstart.rst

```diff
@@ -10,12 +10,18 @@ Installation::
 Get started::
 
     from browser_history.browsers import Firefox
+    from browser_history.utils import *
 
     f = Firefox.fetch()
 
-    histories = f.get()
+    his = f.get()
+
+    all_browsers = get_history()
+
+    all_histories = all_browsers.get()
 
 - ``Firefox`` in the above snippet can be replaced with any of the :ref:`supported_browsers`.
-- ``his`` is a list of ``(datetime.datetime, url)`` tuples.
+- ``his`` is a list of ``(datetime.datetime, url)`` tuples from the firefox browser.
+- ``all_histories`` is a list of ``(datetime.datetime, url)`` tuples from every browser installed on the system.
```

Comment:
Same as README here.

We should probably document this in the contributing guidelines - that README and quickstart have to be updated simultaneously.